job_groups/opensuse_tumbleweed_riscv64.yaml

```
@@ -0,0 +1,40 @@
+############################################################
+#                         WARNING                          #
+#                                                          #
+#               This file is managed in GIT!               #
+#  Any changes via the openQA WebUI will get overwritten!  #
+#                                                          #
+#    https://github.com/os-autoinst/opensuse-jobgroups     #
+#       job_groups/opensuse_tumbleweed_riscv64.yaml        #
+############################################################
+
+---
+
+defaults:
+  riscv64:
+    machine: riscv64_cpu_max
+    priority: 50
+products:
+  opensuse-Tumbleweed-JeOS-for-RISCV-riscv64:
+    distri: opensuse
+    flavor: JeOS-for-RISCV
+    version: Tumbleweed
+scenarios:
+  riscv64:
+    opensuse-Tumbleweed-JeOS-for-RISCV-riscv64:
+      - jeos:
+          settings:
+            EXCLUDE_MODULES: diskusage,libzypp_config
```

I just changed this on o3 (https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/19899 got merged):

```suggestion
            EXCLUDE_MODULES: diskusage,libzypp_config
```

---

https://github.com/os-autoinst/opensuse-jobgroups/pull/501